### ISSUE TITLE
Remove stale bot

### ISSUE THREAD

**Issue by evrardjp** [weaveworks/kured#406]
The stale bot allowed us to keep some interesting issues, while we got the opportunity to clean many things.

I don't think we need the bot anymore (we are managing the issues and PR better now). 
Nowadays it even hurts more than its doing good: it brings friction to issues and pull requests ("Why is this bot closing something that matters to me!").

Should we remove it?

**Comment by dholbach** [comment]
^ haha ... :joy:    the :robot: doesn't like the idea!

**Comment by VedRatan** [comment]
Yes, the stale bot is unused as it may remove the important issues without any confirmation from the repo owner. By the way, can I know that why this question is marked as the good first issue? If its a reviewing question, then you can assign me the issue. @evrardjp

**Comment by VedRatan** [comment]
Yes, the stale bot may remove the important issues without any confirmation from the repo owner. By the way, can I know that why this question is marked as the good first issue? If its a reviewing question, then you can assign me the issue. @evrardjp
